Profile.js

```diff
@@ -0,0 +1,61 @@
+import React, {useState} from 'react';
```

Comment:
We are not using js files at all. we will be going forward with tsx only, hence make changes accordingly